### ISSUE TITLE
Debugger can only run file found on the same drive as the Prolog installation

### ISSUE THREAD

**Issue by omubtcezq** [arthwang/vsc-prolog#21]
If SWI Prolog is installed on one drive (e.g. D:), then running the debugger on a file on another drive (e.g. C:) will look for the file in Prolog's installation drive (D:), and VS Code will error saying the file cannot be found. Error shown:
`Unable to open 'test.pl': File not found`
`(file:///<path-to-file-without-drive-letter>/test.pl).`
Pressing the _Create File_ option creates the file in the correct path, but under the wrong drive (creates the file on D: instead of finding the file on C:).

Adding the the following command-line parameters to switch drive on swipl startup was tried but did not work. Likely due to how the file being debugged is being run by swipl.

`"runtimeArgs": ["-g", "working_directory(_,'C:')"],`

Drive set-up which produced the error:
- SWI-Prolog version 7.6.4 x64-win64
- Prolog installed on drive: D:
- Debugger run on file on drive: C:
- Windows installation on drive: C:

**Comment by arthwang** [comment]
In launch.json, default setting {"configurations":[{... "program": "${file}" ,.." yields the correct file path. ${file} refers to active file.
Please attach your ./vscode/launch.json.

**Comment by lucaciraolo** [comment]
Perhaps a similar issue, I get the following error when trying to run the debugger:
`Unable to open 'underground.pl': File not found (file:///users/luca%20ciraolo/documents/github/ecm2418-prolog-ca2/ca2/underground.pl).`

Could it be a problem with my user name? This is on Windows 10.

My ./vscode/launch.json file is pretty much the default (I only changed the path to executable)
```
{
    // Use IntelliSense to learn about possible attributes.
    // Hover to view descriptions of existing attributes.
    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
    "version": "0.2.0",
    "configurations": [
        {
            "name": "Debug Prolog Files",
            "type": "prolog",
            "request": "launch",
            "runtimeExecutable": "C:\\Program Files\\swipl\\bin\\swipl.exe",
            "runtimeArgs": [],
            "env": {},
            "program": "${file}",
            "cwd": "${fileDirname}",
            "startupQuery": "start",
            "stopOnEntry": true,
            "traceCmds": {
                "continue": [
                    "leap",
                    "l"
                ],
                "stepover": [
                    "skip",
                    "s"
                ],
                "stepinto": [
                    "creep",
                    "c"
                ],
                "stepout": [
                    "up",
                    "u"
                ]
            }
        }
    ]
}
```

**Comment by arthwang** [comment]
Yeah, seems the issue is caused by the space in your name. Don't contain spaces in your working path.   That seems a vscode debugger's bug.

**Comment by jy95** [comment]
I got the "Cannot lint the prolog file. The Prolog executable was not found. Use the 'prolog.executablePath' setting to configure" message because Prolog in stored on C: ( "C:\\Progs\\swipl\\bin\\swipl.exe") whereas my code is on D:
